src/main/resources/assets/seedqueue/lang/es_es.json

```diff
@@ -0,0 +1,127 @@
+  "speedrunapi.config.seedqueue.option.resetCooldown.description": "Tiempo de espera mientras una preview no puede ser reseteada, Empieza cuando la preview acaba de empezar a renderizarse.",
+  "speedrunapi.config.seedqueue.option.waitForPreviewSetup": "Esperar al Render de Preview",
+  "speedrunapi.config.seedqueue.option.waitForPreviewSetup.description": "Si está activo, las semillas se enseñarán en la ventana de la Wall cuando estén listas para enseñar la preview.\nPuedes desactivar esta opción si no te importa ver los Chunkmaps mientras la preview está preparandose.",
+
+  "speedrunapi.config.seedqueue.option.bypassWall": "Puentear Ventana de la Wall",
+  "speedrunapi.config.seedqueue.option.bypassWall.description": "Puentea la ventana de la wall poniendote en la próxima semilla generada bloqueada si es posible.",
+  "speedrunapi.config.seedqueue.option.wallFPS": "FPS de la Wall",
+  "speedrunapi.config.seedqueue.option.wallFPS.description": "Límite de fotogramas de la ventana de la wall.",
+  "speedrunapi.config.seedqueue.option.wallFPS.value": "%s fps",
+  "speedrunapi.config.seedqueue.option.wallFPS.value.255": "Sín Límite",
+  "speedrunapi.config.seedqueue.option.previewFPS": "FPS de la Preview",
+  "speedrunapi.config.seedqueue.option.previewFPS.description": "EL límite de fotogramas usados en las previews de los mundos en la Wall.\nEste ajustes es relativo al límite de fotogramas de la Wall, así que si usas Sin Límite como ajuste en la wall, esta opción puede no funcionar correctamente.",
+  "speedrunapi.config.seedqueue.option.previewFPS.value": "%s fps",
+  "speedrunapi.config.seedqueue.option.previewFPS.value.255": "Sín Límite",
+  "speedrunapi.config.seedqueue.option.backgroundPreviews": "Previews en segundo plano",
+  "speedrunapi.config.seedqueue.option.backgroundPreviews.description": "Crea previews de los mundos en segundo plano antes de que se estén renderizando creando una mejor transición cuando cargas una nueva preview en la Wall.",
+  "speedrunapi.config.seedqueue.option.freezeLockedPreviews": "Congelar Previews Bloquadas",
+  "speedrunapi.config.seedqueue.option.freezeLockedPreviews.description": "Al activarlo, las previews de los mundos que hayas bloqueado no generarán mas mundo.",
+  "speedrunapi.config.seedqueue.option.keyBindings": "Combinaciones de Teclas",
+  "speedrunapi.config.seedqueue.option.showAdvancedSettings": "Mostrar Ajustes Avanzados",
+  "speedrunapi.config.seedqueue.option.seedQueueThreadPriority": "Prioridad de Hilos SeedQueue",
+  "speedrunapi.config.seedqueue.option.seedQueueThreadPriority.description": "Cambia la prioridad de los hilos de SeedQueue para generar semillas (5 es la mayor).",
+  "speedrunapi.config.seedqueue.option.serverThreadPriority": "Prioridad Desenfocada",
+  "speedrunapi.config.seedqueue.option.serverThreadPriority.description": "Cambia la prioridad de los hilos de generación de semillas cuando no estás usando la ventana de Minecraft (5 es la mayor).",
```

Comment:
"Sets the thread priority for seeds generating in the background." from the english translation refers to seeds generating in the background, not minecraft in the background. The server thread is the thread for a seed in the queue.

Reply:
Fixed on latest commit. Thanks